### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by LinkTheCoder** [MyFursona-Project/MyFursona#39] [General]
@kuroji-fusky Does MyFursona have a Discord server?

For announcements, bug reporting, questions etc :3

**Comment by ghost** [comment]
No sadly, I'm not even sure if @VulpoTheDev is considering making one, but we would probably consider making one eventually if we have dozens of devs on board, so yeah :3

We're still keeping this project on the down low for now until we have a public stable release

**Comment by VulpoTheDev** [comment]
As of right now we don't have a discord server for the sake of we don't want to make everything known to public (mainly to avoid stress and the need to rush things out). We will make one once we release a working version of the website with minimal/core functionality

**Comment by LinkTheCoder** [comment]
OK 👍

**Comment by VulpoTheDev** [comment]
That is a good question so I'll be pinning this. If you ever want to reach me and kuro out mines is ozzythedev and kuro is kurojifusky

**Comment by VulpoTheDev** [comment]
Actually if anything if you want I could invite your to the MF group chat where we discuss what's our next move if that's something you'd be interested in
